addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -164,7 +171,30 @@ if (isMultiplayer) then {
             private _addons = GVAR(clientAddons) - GVAR(serverAddons);
 
             if (_addons isNotEqualTo []) then {
-                private _errorMsg = format ["Client/Server Addon Mismatch. Client has additional addons: %1. Server modDir: %2", _addons, GVAR(serverSource)];
```

Comment:
I'm not a fan of losing info on additional addons above 3. I thin we should make 2 error messages: The current one can be used in the UI, but the other one should contains all additional addons and be printed in full in the log.